assets/embauche.html

```diff
@@ -50,7 +50,16 @@
 					<option>mois</option>
 					<option disabled>jour</option>
 					<option disabled>heure</option>
-				</select>.
+				</select>
+
+				sur la commune de
+				<input name="code_postal_entreprise" id="code_postal_entreprise" class="medium-width centered"
```

Comment:
:confused: and FF doesn't support `width: attr(length em)` as a fallback either  :disappointed: 

I'm in favour of styling `#code_postal_entreprise`. Right now, we have only one example, thus no need for a class, and if another input is `medium-width` perhaps it won't be 5 characters long.